agentops/llms/providers/cohere.py

```diff
@@ -3,11 +3,11 @@
 from typing import Optional
 
 from .instrumented_provider import InstrumentedProvider
-from ..event import ActionEvent, ErrorEvent, LLMEvent
-from ..session import Session
-from ..log_config import logger
-from agentops.helpers import get_ISO_time, check_call_stack_for_agent_id
-from ..singleton import singleton
+from ...event import ActionEvent, ErrorEvent, LLMEvent
+from ...session import Session
+from ...log_config import logger
+from ...helpers import get_ISO_time, check_call_stack_for_agent_id
```

Comment:
please use absolute imports when not importing from within the same module portable module ( out of scope imports )

Reply:
rule of thumb is if import already goes out of its directory tree (`..`) is a no-go
This is acceptable in rare circumstances such as tests suites / fixtures, etc.